data/locale/en-US.ini

```diff
@@ -93,7 +99,13 @@ NVENC.Preset.Lossless="Lossless"
 NVENC.Preset.LosslessHighPerformance="Lossless High Performance"
 NVENC.RateControl="Rate Control Options"
 NVENC.RateControl.Mode="Mode"
-NVENC.RateControl.Mode.CQP="Constant Quantization Parameter"
+NVENC.RateControl.Mode.Description="Rate control mode selection"
+NVENC.RateControl.Mode.CQP.Description="A flat compression ratio with no regard for bit rates.\nThis yields the highest quality-per-bitrate."
```

Comment:
`\nThis yields the highest quality-per-bitrate.` since this depends on the users settings, don't include it.